codersdk/users.go

```diff
@@ -54,9 +54,8 @@ type ReducedUser struct {
 	UpdatedAt   time.Time `json:"updated_at" table:"updated at" format:"date-time"`
 	LastSeenAt  time.Time `json:"last_seen_at" format:"date-time"`
 
-	Status          UserStatus `json:"status" table:"status" enums:"active,suspended"`
-	LoginType       LoginType  `json:"login_type"`
-	ThemePreference string     `json:"theme_preference"`
```

Comment:
_Technically_ this is a breaking change. We should leave it but mark the field as deprecated in favour of `UserAppearanceSettings.ThemePreference.`

Reply:
I think the practical likelihood of anyone but us using this field is zero, but I'd be ok to put it back as long as we just leave it as an empty string.